package.json

```diff
@@ -73,9 +73,13 @@
   "dependencies": {
     "arr-flatten": "^1.0.1",
     "ava-init": "^0.1.0",
-    "babel-core": "^5.8.23",
-    "babel-plugin-espower": "^1.1.0",
-    "babel-runtime": "^5.8.29",
+    "babel-core": "6.0.14",
```

Comment:
Yep, at the time, latest was `6.0.18` which I couldn't get to work at all. That is why it is pinned.
They are on `6.3.?` now, so this likely isn't necessary anymore.